### DISCUSSION TITLE
how register mitmproxy as a CA

### DISCUSSION THREAD

**Discussion by EdmundsEcho** [mitmproxy/mitmproxy#6630] [Q&A]
I'm new to using this tool.  I have things working using http requests.  However, I'm getting a "The client does not trust the proxy's certificate" error when I call `curl https://archive.is/`.  I get that mitmproxy is a registered CA.  If memory servers, macos has a list of trusted CAs.  Is mitmproxy on that list?

The error in the stdout: `SSL certificate problem: unable to get local issuer certificate`. The previously quoted error is that in the logs.

Finally, am I missing something else to troubleshoot this issue?

Note: I'm using my `.curlrc` file to specify the mitmproxy proxy listening on port 8080

**Comment by emanuele-em** [comment]
If you are on macOS you can trust the certificate that you previously downloaded following these steps:
1. open keychain app 
<img width="400" alt="image" src="https://github.com/mitmproxy/mitmproxy/assets/100081325/7c4f8a36-7c93-4a5a-a7e6-8835c77b34fb">

2. find the mitmproxy certificate in the list:
<img width="281" alt="image" src="https://github.com/mitmproxy/mitmproxy/assets/100081325/e6616135-1b71-4147-bc7b-cb5cbbd9073e">


3. double click on it and click on **trust** menu: 
<img width="506" alt="image" src="https://github.com/mitmproxy/mitmproxy/assets/100081325/4a5125a3-30ae-4394-9e32-f79fa78a500c">

4. Select **Always trust** from the dropdown menu

**Comment by EdmundsEcho** [comment]
I installed mitmproxy using homebrew.  I'm sharing this because I did not see separate instructions for downloading a certificate.  The keychain app is not pulling up an entry for mitmproxy.
